types/kraken-js/kraken-js-tests.ts

```diff
@@ -0,0 +1,32 @@
+import * as express from 'express';
+import * as kraken from 'kraken-js';
```

Comment:
Since `kraken` is a function, it should be `import kraken = require('kraken-js');` here

Reply:
Thanks, I've missed that detail somehow. Fixed.